### PR TITLE
bug/403_service_path_slash

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue #403 
* 100% unit tests passed:
```
Results :
Tests run: 49, Failures: 0, Errors: 0, Skipped: 0
```
* (unofficial) e2e tests passed:
```
> show collections
frb_/zzz_serv_path
frb_/zzz_serv_path.aggr
system.indexes
```
* Assignee @iariasleon

### CODE REVIEW

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionMongoSink.java [hunk 1]
```diff
@@ -51,7 +51,12 @@ void persist(Map<String, String> eventHeaders, NotifyContextRequest notification
         // get some header values
         Long recvTimeTs = new Long(eventHeaders.get("timestamp"));
         String fiwareService = eventHeaders.get(Constants.HEADER_SERVICE);
-        String fiwareServicePath = eventHeaders.get(Constants.HEADER_SERVICE_PATH);
+        String[] fiwareServicePaths = eventHeaders.get(Constants.HEADER_SERVICE_PATH).split(",");
```
Comment (iariasleon) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionMongoSink.java]: Is a service path list now?

File: doc/design/OrionMongoSink.md [hunk 2]
```diff
@@ -54,23 +54,28 @@ Assuming `mongo_username=myuser` as configuration parameter, then `OrionMongoSin
     > show databases
```
Comment (fgalan) [inline doc/design/OrionMongoSink.md]: CHANGES NEXT RELEASE entry missing?

File: src/main/java/com/telefonica/iot/cygnus/sinks/OrionMongoSink.java (hunk 1 above)
Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/sinks/OrionMongoSink.java]: This is hard to explain, but I'll try :) Originally, an Orion notification, which may contain serveral context elements, has a unique fiware-servicePath for all of them. Nevertheless, internally to Cygnus this common fiware-servicePath may be discarded in favor of an alternative fiware-servicePath for each context element if the pattern-based grouping mechanisms is configured to do that. For instance, a notification may contain the "speed" value for a "car.1" of type "car" and the "speed" value for a "car.mycar" of type "car"; the common fiware-servicePath is "/4wheels". If the following rules are configured:

* "all cars called as *.d will go to a destination called numeric_cars in a fiware-servicePath called /4wheels_numeric_cars"
* "all cars called as *.D will got to a destination called other_cars in a fiware-servicePath called /4wheels_other_cars"

Then we have the origianl fiware-servicePath transformed in a list of two elements, containing ["/4wheels_numeric_cars","/4wheels_other_cars"].

File: doc/design/OrionMongoSink.md (hunk 2 above)
Comment (frbattid) [inline doc/design/OrionMongoSink.md]: Missing as usual :)

Comment (frbattid) [inline doc/design/OrionMongoSink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/0119928dc774062e17071dc7823944f3cc04fb19

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**